### PR TITLE
Fix cuttlefish

### PR DESCRIPTION
**Pull request by davecromberge**
Same issue as https://github.com/tsloughter/rebar3_cuttlefish/issues/7, upgrade rebar3 to newer version as suggested.

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -9,7 +9,7 @@
 {deps,
  [
   lager,
-  {cuttlefish, ".*", {git, "https://github.com/tsloughter/cuttlefish.git", {branch, "develop"}}},
+  {cuttlefish, "2.0.7"},
```
Comment (Licenser) [inline rebar.config]: this is not needed any more, it shouldn't be included or it can conflict with the cuttlefish plugin

**Comment by Licenser** [comment]
Hi thanks for this I haven't gotten around to updating the FE yet, sorry.

A few points:
1) (see inline comment)
2) `{copy, "_build/default/bin/cuttlefish", "bin/cuttlefish"},` needs to be gone from the relx section
3) the schema sources should be `prod` not `default` as a base

**Comment by davecromberge** [comment]
@Licenser thanks for the feedback, I have addressed all but the last point.  Can you give a slightly more detailed description?

**Comment by Licenser** [comment]
Awesome thanks!

the `{template, "_build/default/lib/dqe/priv/dqe.schema", "share/schema/01-dqe.schema"}` needs to read
`{template, "_build/prod/lib/dqe/priv/dqe.schema", "share/schema/01-dqe.schema"}` since when build with prod it might not be available in default.

**Comment by davecromberge** [comment]
Do you mean to use `\{\{build_dir\}\}/lib/` instead?

**Comment by Licenser** [comment]
not sure if that gets substituted, does it?

**Comment by davecromberge** [comment]
@Licenser it's not something automatic, needed some extra config in `vars.config`

**Comment by Licenser** [comment]
they don't apply top the path I think, it only applies to the files.

But what I mean is it needs to read:
`{template, "_build/prod/lib/dqe/priv/dqe.schema", "share/schema/01-dqe.schema"}`
not 
`{template, "_build/default/lib/dqe/priv/dqe.schema", "share/schema/01-dqe.schema"}`

**Comment by davecromberge** [comment]
I made the change as you suggest, but it only works when I do a `rebar3 as prod release`, in the default profile it no longer would work.  So in order to get the release to work under both profiles, I introduced the build_path config, which detects the profile.  I may be mistaken, because the rebar3 stuff is still a bit cryptic at this point to me.

**Comment by Licenser** [comment]
awesome, and wow that's icky but I see why it's needed.

**Comment by davecromberge** [comment]
Sure @Licenser I agree, it is icky, I can't take the credit though as my colleague found it out when our CI stopped working for dalmatinerdb.

**Pull request merged**